### PR TITLE
adding SQL DB migration cmdlets to AZ.DataMigration

### PR DESCRIPTION
**Pull request by vasundhraManhas**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1162
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by vasundhraManhas** [comment]
@isra-fel  The swagger has been merged :  https://github.com/Azure/azure-rest-api-specs/pull/18213

File: src/DataMigration/DataMigration.Autorest/readme.md [hunk 1]
```diff
@@ -30,18 +30,22 @@ For information on how to develop for `Az.DataMigration`, see [how-to.md](how-to
 > see https://aka.ms/autorest
 
 ``` yaml
-branch: 7086ee861c3a6196bb98f8b327af11d03e545a05
+branch: dev-datamigration-2022-03-30-preview
```
[the remark this answers is not in the document]
Comment (vasundhraManhas) [reply src/DataMigration/DataMigration.Autorest/readme.md]: done

**Review by isra-fel — APPROVED** [review]
Looks good to me. But note that the skipped test cases must be enabled prior to Az.DataMigration module GA.

**Comment by vasundhraManhas** [comment]
> Looks good to me. But note that the skipped test cases must be enabled prior to Az.DataMigration module GA.

Thank you so much!

**Pull request merged**